url.js

```diff
@@ -150,7 +149,7 @@ Url.prototype.parse = function (url, parseQueryString, slashesDenoteHost) {
       if (simplePath[2]) {
         this.search = simplePath[2];
         if (parseQueryString) {
-          this.query = querystring.parse(this.search.substr(1));
+          this.query = Object.fromEntries(new URLSearchParams(this.search.substr(1)));
```

Comment:
`Object.fromEntries` also won't work in our supported browsers, but at least this can come from https://npmjs.com/object.fromentries